Example/HorizonCalendarExample/HorizonCalendarExample/DemoPickerViewController.swift

```diff
@@ -108,7 +108,7 @@ extension DemoPickerViewController: UITableViewDelegate {
     let demoViewController = demoDestinations[indexPath.item].destinationType.init(
       monthsLayout: monthsLayoutPicker.selectedSegmentIndex == 0
         ? .vertical(pinDaysOfWeekToTop: false)
-        : .horizontal(monthWidth: min(view.bounds.width - 128, 512)))
```

Comment:
subtracting 128 from 375 just doesn't leave enough room for 7 days when there's horizontal spacing between individual days.

Reply:
The good news is the internal assert works to let you know the layout isn't valid 😄